### DISCUSSION TITLE
Rookie mistakes? or device compatibility problem?

### DISCUSSION THREAD

**Discussion by sky2739** [andrei-tatar/node-red-contrib-smartnora#106] [Q&A]
I'm a beginner in home automation. So I really don't understand any codes or flows at all.
Been reading guides and articles of node-red these two days.
Maybe some of my questions are pretty obvious, but I just don't know the principle behind everything...
Hopefully I will learn how to make some simple automated tasks and routines in the next few weeks.

I installed Node-Red on my Synology NAS Docker, and followed the README, installed "install node-red-contrib-smartnora", linked Smart NORA and Google Home.

I think I didn't mess up the flow, 
But the nora node doesn't really recognize my device, is there another step I missed?

I was testing a power outlet (TP-LINK P100), so I made a outlet flow and deployed it.
And... in my Google Home App, it added a new outlet, instead of controlling my P100 outlet.

I'm unsure if this is right? 
Maybe I have to somehow sync the nora outlet with my P100 on Google Home?

Or SmartNora doesn't support my device? So it can't function normally?

**Comment by andrei-tatar** [comment]
Scope of Smart NORA is not to detect and sync your home devices or to control existing Google Home devices.
The scope is to define your own devices that you can control via Google Assistant/Home. For example you can create your own outlet or your own integration for an existing outlet.

So if you already have the P100 outlet added in Google Home, you don't need Smart NORA and you rely on TpLink's service for Google Home integration.

You need Smart NORA if:
1. you don't want to use TpLink's integration (or some other deprecated service)
2. you create your own devices or you have devices that don't have a cloud service - so you can develop your own integration between Nora and your device.
